packages/remediations/.gitignore

```diff
@@ -7,3 +7,4 @@
 /esm
 /cjs
 /*.css
+*.txt
```

Comment:
was this change intended?

Reply:
Yup, the build now creates `.txt` files containing some license info from minimize plugin.